### PR TITLE
Add Allow Origin and pre-flight options to bypass CORS

### PR DESCRIPTION
**Pull request by Chroq**
## Description

I add some headers on the request to allow all origins and pre-flight options. It's just for the POC, it's not a real production configuration. 

Trello : [https://trello.com/c/zqtlVJDr/6-1-as-norbert-i-want-to-know-when-any-of-the-players-have-won](url)

### CODE REVIEW

File: src/api/handler/handler.go [hunk 1]
```diff
@@ -9,15 +9,32 @@ import (
...
+	prepareHeaders(&responseWriter)
 	responseWriter.WriteHeader(http.StatusOK)
-	responseWriter.Header().Set("Content-Type", "application/json")
+
 	io.WriteString(responseWriter, response)
 }
 
+func prepareHeaders(writer *http.ResponseWriter) {
+	(*writer).Header().Set("Content-Type", "application/json")
+	(*writer).Header().Set("Access-Control-Allow-Origin", "*")
```
Comment (djhi) [inline src/api/handler/handler.go]: Why do you need to add it twice, here and at L33 ?

File: src/api/handler/handler.go [hunk 2]
```diff
@@ -9,15 +9,32 @@ import (
...
 
+	setupResponse(&responseWriter)
+	if (*request).Method == "OPTIONS" {
+		return
+	}
+
+	currentGame := getGameByRequest(request)
 	response := fmt.Sprintf(`{"isWon": %t}`, game.IsWinningGame(currentGame))
 
+	prepareHeaders(&responseWriter)
```
Comment (alexisjanvier) [inline src/api/handler/handler.go]: Yes, @djhi  is right. No need to use `prepareHeaders` here, it's already done line 13 with the `setupResponse`, right?

File: src/api/handler/handler.go (hunk 1 above)
Comment (Chroq) [reply src/api/handler/handler.go]: Indeed, fixed ! TY

File: src/api/handler/handler.go (hunk 2 above)
Comment (Chroq) [reply src/api/handler/handler.go]: Yes, fixed ! TY

**Pull request merged**